packages/create-sitecore-jss/src/templates/nextjs-sxa/package.json

```diff
@@ -1,5 +1,9 @@
 {
   "dependencies": {
     "bootstrap": "^5.1.3"
```

Comment:
Do we still need bootstrap (here and in `_app.tsx`)?

Reply:
Yes, the bootstrap uses with our styles.